<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Imagenes_mdicas</class>
 <widget class="QMainWindow" name="Imagenes_mdicas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frameCargar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>761</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnCargarImagenMedica">
       <property name="font">
        <font>
         <family>Copperplate Gothic Bold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 255);</string>
       </property>
       <property name="text">
        <string>Cargar Imagen médica</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnProcesarImagen">
       <property name="font">
        <font>
         <family>Copperplate Gothic Bold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 255);</string>
       </property>
       <property name="text">
        <string>Procesar Imagen</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frameCortes">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>921</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="Line" name="line1">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>20</width>
       <height>241</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line2">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>20</width>
       <height>241</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelAXIAL">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Copperplate Gothic Bold</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>AXIAL</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCORONAL">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Copperplate Gothic Bold</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CORONAL</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSAGITAL">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>20</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Copperplate Gothic Bold</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SAGITAL</string>
     </property>
    </widget>
    <widget class="QSlider" name="Slideraxial">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>20</width>
       <height>181</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="Slidercoronal">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>20</width>
       <height>181</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="Slidersagital">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>50</y>
       <width>20</width>
       <height>181</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frameAxial">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>251</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lblImagenAxial">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frameCoronal">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>271</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lblImagenCoronal">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>251</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frameSagital">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>50</y>
       <width>261</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lblImagenSagital">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="lblTituloImagenesmedicas">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate Gothic Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IMÁGENES MÉDICAS</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblArchivocargado">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate Gothic Bold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Archivo Cargado: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGuardarImagen">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate Gothic Bold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>Guardar Imagen</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEstado">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>540</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate Gothic Bold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Estado: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lblrtaEstado">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>540</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>____________</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblrtaArchivoC">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>140</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>_________________</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDatosP">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>600</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate Gothic Bold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DATOS DEL PACIENTE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInfoPaciente">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>640</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate Gothic Bold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>Ver Información</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>954</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
